src/cmd/manage_nsfs.js

```diff
@@ -168,18 +178,26 @@ async function update_bucket_object(config_root, target) {
         print_account_usage();
     }
     const data = _.merge({}, source, target);
-    data.path = source.path;
+    // TODO: ask Naveen why this line was added
+    //data.path = source.path;
```

Comment:
Hey @naveenpaul1 I saw that that source path is always taken, why is that? a user won't be able to update path according to this line

Reply:
@romayalon Idea is source(new json schema for update) should be able to update all properties in the existing bucket,  `_.merge()`  was not updating the `path` on its own, So I have to do it in code explicitly.

Reply:
but this line - data.path = source.path; won't update the data with a new path, it takes the source and not the new path. or am I missing something?
and thanks i'll update the print

Reply:
@romayalon got your point and you are correct, This code takes the same path from the existing bucket schema not using new path. It should be `target.path`.